client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { Button } from '@automattic/components';
+import { isEnabled } from '@automattic/calypso-config';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'calypso/components/gridicon';
+import {
+	emailManagementManageTitanAccount,
+	emailManagementTitanControlPanelRedirect,
+} from 'calypso/my-sites/email/paths';
+import { TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL } from 'calypso/lib/titan/constants';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import {
+	EMAIL_ACCOUNT_TYPE_TITAN_MAIL,
+	EMAIL_WARNING_SLUG_UNUSED_MAILBOXES,
+} from 'calypso/lib/emails/email-provider-constants';
+
+class EmailPlanWarnings extends React.Component {
+	static propTypes = {
+		domain: PropTypes.object.isRequired,
+		emailAccountType: PropTypes.string.isRequired,
+		warnings: PropTypes.array,
+	};
+
+	renderWarningCTAForTitanUnusedMailboxes() {
+		const { selectedSiteSlug, domain, currentRoute, translate } = this.props;
+
+		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
+		const controlPanelUrl = showExternalControlPanelLink
+			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } )
+			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } );
+
+		return (
+			<Button
+				compact
+				primary
+				href={ controlPanelUrl }
+				target={ showExternalControlPanelLink ? '_blank' : null }
+			>
+				{ translate( 'Activate Mailboxes' ) }
+				{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
+				{ showExternalControlPanelLink && <Gridicon icon="external" size={ 16 } /> }
```

Comment:
It also looks like the 16 size isn't really being honoured -- on Chrome I see the icon at 18px by 18px via CSS. If that's the case, it may be better to drop the `size` attribute here.